metadata.json

```diff
@@ -5,7 +5,7 @@
     "source": "git://github.com/puppetlabs/puppetlabs-puppetdb.git",
     "project_page": "http://github.com/puppetlabs/puppetlabs-puppetdb",
     "author": "puppetlabs",
-    "license": "Apache 2.0",
+    "license": "Apache-2.0",
```

Comment:
This gets rid of the warning "Unrecognized license in metadata." on the Forge and allows the metadata-json-lint to exit without an error.

https://forge.puppetlabs.com/puppetlabs/puppetdb/scores